src/processor.py

```diff
@@ -287,97 +285,50 @@ def process_csv_corpus(file_path, output_dir_path, corpus_name, text_col_name,
     Returns:
         None
     """
-    df = read_csv_corpus(file_path, sep, names)
+    if corpus_name in ('Alpaca-gn-gpt4','Alpaca-gn-gpt3.5','gn-multi-affective-alpaca','mala-monolingual-split','cc100_gn'):
+        df = read_csv_corpus(file_path, sep, names, drop_incomplete_records=False)
+    else:
+        df = read_csv_corpus(file_path, sep, names)
+
     if df.shape[0] > 0:
+
         print(f'Processing corpus: {"/".join(file_path.split("/")[-2:])}')
         report_dict = get_report_dict()
         corpus_file_name = file_path.split('/')[-1]
         data = []
-        if corpus_name == 'multi-wiki-qa':
-            for _, row in df.iterrows():
-                context, question, answer = row[text_col_name[0]], row[text_col_name[1]], row[text_col_name[2]]
-                text = ""
-                if isinstance(context, str):
-                    text += context
-                if isinstance(question, str):
-                    text += f"\n\n{question}"
-                if isinstance(answer, str):
-                    try:
-                        answer_dict = ast.literal_eval(answer)
-                        text += f"\n\n{' '.join(answer_dict["text"])}"
-                    except Exception:
-                        pass                
-                if not text:
-                    continue
+        contador = 0
+        for _, row in df.iterrows():
+            if corpus_name in ('Alpaca-gn-gpt4','Alpaca-gn-gpt3.5','gn-multi-affective-alpaca'):
```

Comment:
The condition includes a repetitive clause. If the tuple is used more than once, set it to a variable instead of repeating it.